packages/filesystem/src/browser/file-tree/file-tree-widget.tsx

```diff
@@ -315,4 +315,8 @@ export class FileTreeWidget extends CompressedTreeWidget {
         return inflated;
     }
 
+    protected override getDepthPadding(depth: number): number {
+        // add additional depth so file nodes are rendered with padding in relation to the top level root node.
+        return (depth + 1) * this.props.leftPadding;
```

Comment:
```suggestion
        return super.getDepthPadding(depth + 1);
```

Reply:
Hi @paul-marechal , thanks a lot for the suggestion.  I have updated the code accordingly.